### PR TITLE
CloudFormation Resource Specification 20.3.0

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*
- Update specs to 20.3.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by PatMyron** [comment]
another instance of https://github.com/awslabs/goformation/issues/300
[`AssertionError: False != True : Specified property type AWS::DataBrew::Recipe. not found for property Parameters`](https://travis-ci.com/github/aws-cloudformation/cfn-python-lint/jobs/439412932#L296)
```json
    "AWS::DataBrew::Recipe.Action": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-databrew-recipe-action.html",
      "Properties": {
        "Operation": {
          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-databrew-recipe-action.html#cfn-databrew-recipe-action-operation",
          "UpdateType": "Mutable",
          "Required": true,
          "PrimitiveType": "String"
        },
        "Parameters": {
          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-databrew-recipe-action.html#cfn-databrew-recipe-action-parameters",
          "UpdateType": "Mutable",
          "Required": false
        }
      }
    },
```
https://schema.cloudformation.us-east-1.amazonaws.com/aws-databrew-recipe.json

File: scripts/update_specs_from_pricing.py [hunk 1]
```diff
@@ -49,6 +49,8 @@
     'US West (Oregon)': 'us-west-2',
     'US West (Los Angeles)': 'us-west-2',
     'US West (Verizon) - San Francisco Bay Area': 'us-west-2',
+    'US East (Verizon) - Miami': 'us-east-1',
+    'US East (Verizon) - Dallas': 'us-east-1',
```
Comment (PatMyron) [inline scripts/update_specs_from_pricing.py:53]: Wondering if it'd be worth supporting patterns if this list keeps growing
```suggestion
    'US West (Verizon) - .*': 'us-west-2',
    'US East (Verizon) - .*': 'us-east-1',
```

Comment (kddejong) [reply scripts/update_specs_from_pricing.py:53]: was starting to debate that too.  I wonder if we can just drop the verizon data?  I don't think it adds anything either.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Auto generated files for testing
+test/fixtures/templates/bad/limit_numbers.yaml
+test/fixtures/templates/bad/limit_size.yaml
```
Comment (PatMyron) [inline .gitignore:3]: good catch 👍  should've added this in https://github.com/aws-cloudformation/cfn-python-lint/pull/1747 but too accustomed to testing remotely with Travis

File: scripts/update_specs_from_pricing.py (hunk 1 above)
Comment (PatMyron) [reply scripts/update_specs_from_pricing.py:53]: https://github.com/aws-cloudformation/cfn-python-lint/pull/1793

**Pull request merged**